rasa/core/policies/policy.py

```diff
@@ -83,7 +86,14 @@ def featurize_for_training(
             the :class:`rasa.core.training.data.DialogueTrainingData`
         """
 
-        training_data = self.featurizer.featurize_trackers(training_trackers, domain)
+        # Comment: OUTPUT_PATH being passed through is a temporary solution 
+        # because for now NLU is saved while train;
+
+        # TODO: remove it later; 
+
+        training_data, label_data = self.featurizer.featurize_trackers(
```

Comment:
we should create separate `self.featurizer.featurize_all_actions` (or any other name) that creates `label_data` separately